django_project/certification/views/course_type.py

```diff
@@ -214,10 +214,7 @@ def get_queryset(self):
         """
 
         qs = CourseType.objects.all()
-        if self.request.user.is_staff:
-            return qs
-        else:
-            return qs.filter(creator=self.request.user)
+        return qs
```

Comment:
It is filtered in the front end, so user won't see the edit buttons unless they are the organisation owners. Yes, currently any users can view the everything in organisation details, but won't be able to edit it (read only).
like this one:
![screenshot from 2017-07-31 16-12-07](https://user-images.githubusercontent.com/26101337/28770912-17dec584-760b-11e7-80d7-0c1413adb2ab.png)
![screenshot from 2017-07-31 16-12-17](https://user-images.githubusercontent.com/26101337/28770930-265c3592-760b-11e7-8b92-8284f38e0432.png)
![screenshot from 2017-07-31 16-11-53](https://user-images.githubusercontent.com/26101337/28770913-1983cb28-760b-11e7-93e5-d3949ab4cbb2.png)
![screenshot from 2017-07-31 16-12-26](https://user-images.githubusercontent.com/26101337/28770916-1c0fcefa-760b-11e7-825b-6f58164dce97.png)